nodejs/week2/MealSharingWebsite/src/backend/routes/titleContainsRoute.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express.Router();
+
+const [filterCheapMeals , filterHighCapacityMeals , giveOneRandom] = require('./mealsEvaluator.js');
+
+const addReviews = require('./addReviews.js')
+
+const meals = require('../data/meals.json');
+const reviews = require('../data/reviews.json');
+const reservations = require('../data/reservations.json');
+
+app.get("/api/meals?" , (req , res)=>{
+    // res.send('ku' + req.query.title);
+    if(req.query.title)         res.send(meals.filter ( meals => meals.title.toLowerCase().includes(req.query.title) ));
+    else if(req.query.createdafter)  res.send(meals.filter ( meals => meals.createdAt >= req.query.createdafter));
```

Comment:
You might want to change the format of the dates in your database to for example: 2019-04-05.